docs/Migrations.md

```diff
@@ -0,0 +1,22 @@
+# Diesel Migrations
+
+## Pre-requisites
+
+* `cargo install diesel_cli --no-default-features --features postgres`
+
+## Generating new migrations
+
+* `cd components/[builder-originsrv, builder-jobsrv, builder-sessionsrv]`
+* `diesel migration generate <your migration name>`
+
+This will generate something like
+
+```
+Creating migrations/20160815133237_create_posts/up.sql
+Creating migrations/20160815133237_create_posts/down.sql
+```
+
+You can then edit these files to create and revert your migration steps.
+`up.sql` should create your migration
+`down.sql` should revert your migration
+
```

Comment:
Section/text on how to test the up/down migrations?

Reply:
I created these files and I think I edited them properly, but it doesn't look like the migration was run when I started Builder back up.